composer.json

```diff
@@ -1,18 +1,18 @@
 {
     "require": {
         "bcremer/sculpin-commonmark-bundle": "^0.5.0",
-        "bcremer/sculpin-less-bundle": "^0.2.0",
+        "bcremer/sculpin-less-bundle": "0.3.0",
         "dflydev/embedded-composer": "^1.0@dev",
         "ivopetkov/html5-dom-document-php": "^1.1.0",
         "janbuecker/sculpin-meta-navigation-bundle": "^0.9",
         "league/commonmark": "^1.3.0",
-        "mavimo/sculpin-redirect-bundle": "@dev",
-        "sculpin/sculpin": "2.1.2",
-        "symfony/config": "2.8.52",
-        "symfony/dependency-injection": "2.8.52",
-        "symfony/event-dispatcher": "2.8.52",
-        "symfony/http-kernel": "2.8.52",
-        "twig/twig": "1.44.7"
+        "sculpin/sculpin": "3.2.0",
+        "symfony/config": "^4.4.13",
```

Comment:
why not using latest Symfony?

Reply:
This is the requirement of sculin symfony versions. 
They should match.